src/modules/dashboard/dashboard.services.ts

```diff
@@ -5,162 +5,163 @@ import {
+//     updated_time: '1/31/2022',
+//     status: 'Stim',
+//     assigned_to: 'Domainer',
+//     department: 'Konklux',
+//   },
+//   {
+//     id: 7,
+//     raised_by: 'Lyndsie',
+//     created_at: '3/20/2022',
+//     title: "Gone Fishin'",
+//     updated_time: '12/20/2021',
+//     status: 'Voyatouch',
+//     assigned_to: 'Veribet',
+//     department: 'Toughjoyfax',
+//   },
+//   {
+//     id: 8,
+//     raised_by: 'Clemmy',
+//     created_at: '9/6/2022',
+//     title: 'Emma',
+//     updated_time: '7/25/2022',
+//     status: 'Y-find',
+//     assigned_to: 'Mat Lam Tam',
+//     department: 'Tampflex',
+//   },
+//   {
+//     id: 9,
+//     raised_by: 'Kata',
+//     created_at: '7/26/2022',
+//     title: 'Play it to the Bone',
+//     updated_time: '11/30/2021',
+//     status: 'Keylex',
+//     assigned_to: 'Cardguard',
+//     department: 'Cookley',
+//   },
+//   {
+//     id: 10,
+//     raised_by: 'Worth',
+//     created_at: '5/14/2022',
+//     title: 'Duck, You Sucker (a.k.a. Fistful of Dynamite, A) (Giù la testa)',
+//     updated_time: '12/13/2021',
+//     status: 'Temp',
+//     assigned_to: 'Kanlam',
+//     department: 'Asoka',
+//   },
+//   {
+//     id: 11,
+//     raised_by: 'Aurea',
+//     created_at: '11/24/2021',
+//     title: '#chicagoGirl: The Social Network Takes on a Dictator',
+//     updated_time: '4/23/2022',
+//     status: 'Stringtough',
+//     assigned_to: 'Y-Solowarm',
+//     department: 'Vagram',
+//   },
+//   {
+//     id: 12,
+//     raised_by: 'Emalee',
+//     created_at: '1/7/2022',
+//     title: 'Times Square',
+//     updated_time: '2/1/2022',
+//     status: 'Stim',
+//     assigned_to: 'Stronghold',
+//     department: 'Alpha',
+//   },
+//   {
+//     id: 13,
+//     raised_by: 'Reamonn',
+//     created_at: '9/30/2021',
+//     title: 'Christopher Strong',
+//     updated_time: '5/16/2022',
+//     status: 'Tresom',
+//     assigned_to: 'Flexidy',
+//     department: 'Stim',
+//   },
+//   {
+//     id: 14,
+//     raised_by: 'Randy',
+//     created_at: '8/15/2022',
+//     title: 'Zu: Warriors from the Magic Mountain (Xin shu shan jian ke)',
+//     updated_time: '1/6/2022',
+//     status: 'Stronghold',
+//     assigned_to: 'Subin',
+//     department: 'Stronghold',
+//   },
+//   {
+//     id: 15,
+//     raised_by: 'Lonny',
+//     created_at: '6/28/2022',
+//     title: 'Room in Rome (Habitación en Roma)',
+//     updated_time: '12/21/2021',
+//     status: 'Cookley',
+//     assigned_to: 'Andalax',
+//     department: 'Konklab',
+//   },
+// ];
 
-export const getRequestList = (
-  queryParams: IFetchComplaintListRequest
-): Promise<GetRequestsListResponse> => {
-  return Promise.resolve({ message: 'success', data: mockData });
-  // return get({ path: '/complaints', ...queryParams });
+export const getRequestList = (path:string,params: IFetchComplaintListRequest) => {
+  //dummy
+  // return Promise.resolve({ message: 'success', data: mockData });
```

Comment:
remove commented code & { } brackets and return keyword 
we can use arrow function as 
export const getRequestList = (path:string,params: IFetchComplaintListRequest) =>get({ path,queryParams:params });